config.go

```diff
@@ -19,21 +19,21 @@ type Config struct {
 type NativeSystemConfig map[string]any
 
 type LayerServiceConfig struct {
+	Custom                map[string]any `json:"custom"`
 	ServiceName           string         `json:"service_name"`
-	Port                  string         `json:"port"`
+	Port                  json.Number    `json:"port"`
```

Comment:
the json.Number type will accept both "5000" and 5000 as values. when accessing the value, users must decide which type they want (e.g. json.Number("5000").String())